hBank/src/main/java/animalPark/animals/Wolf.java

```diff
@@ -0,0 +1,11 @@
+package animalPark.animals;
+
+import animalPark.AnimalInterface;
+import animalPark.Canine;
+
+public class Wolf extends Canine implements AnimalInterface {
```

Comment:
For this and other classes - Dog, Cat and Lion.
How we can make sure that other classes can't extends these classes any further?